main.py

```diff
@@ -117,11 +118,17 @@ def initialise_configurations():
 
 def set_test_flags(requested_list):
     if not requested_list:
-        requested_list = test_flags
+        requested_list = [
+            key
+            for (key, value) in test_flags.items()
+            if value
+        ]
 
     for item in test_flags:
         if item not in requested_list:
             test_flags[item] = False
+        else:
+            test_flags[item] = True
```

Comment:
For a falsy `request_list` the loop will just set each entry of `test_flags` to what it already is.

Reply:
I've done a bit better. 

If no request list is provider, I didn't loop through the test_flags and use its default value. Otherwise, i've loop the test_flags setting the item values equals true if it is in the requested_list, false otherwise.